docker/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Usage:
+# // To build and publish the Docker image to the internal container registry.
+# $ DESTINATION=internal INTERNAL_CONTAINER_REGISTRY=<internal_registry> ./release.sh
+# // To publish the Docker image to the public container registry.
+# $ DESTINATION=public ./release.sh
+set -ex
+
+PUBLIC_CONTAINER_REGISTRY='gcr.io/stackdriver-agents/stackdriver-logging-agent'
+
+source VERSION
```

Comment:
What is this used for? (Add a comment if it's still needed.) Thanks.

Reply:
+1 on the comment. However, unless we plan to generate the `VERSION` file automatically, you might as well put those variable settings in the script directly. WDYT?